tests/core/tests/test_resources.py

```diff
@@ -1327,6 +1336,7 @@ def test_widget_from_django_field_for_ArrayField_returns_SimpleArrayWidget(self)
         res = resource.widget_from_django_field(f)
         self.assertEqual(widgets.SimpleArrayWidget, res)
 
+
```

Comment:
most insightful comment/question ever: is this extraneous whitespace?

Reply:
Good question :smile: 
I added that because PyCharm complained about that line with:

> PEP 8: E305 expected 2 blank lines after class or function definition, found 1

So I just added it to tidy up whilst editing those lines.

Reply:
Weird. Well I guess we trust Pep 8 ;)